.prettierignore

```diff
@@ -0,0 +1,2 @@
+.cache/*
+public/*
```

Comment:
Might be a good idea to add the `node_modules` directory here so `prettier` doesn't try to check the `node_modules` directory at all.

Reply:
Done.